### PR TITLE
Corrected hyperlink address in help manual -h, -c option by removing the unnecessary bracket.

### PR DESCRIPTION
**Pull request by creaturenex**
### Motivation / Background

This Pull Request has been created because of issue #45988

### Detail

This Pull Request corrects the link address to access the css-bundling readme.

### Additional information

Simple fix, needs to be tested. 

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [x] There are no typos in commit messages and comments.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Feature branch is up-to-date with `main` (if not - rebase it).
* [x] Pull request only contains one commit for bug fixes and small features. If it's a larger feature, multiple commits are permitted but must be descriptive.
* [ ] Tests are added if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.
* [ ] PR is not in a draft state.
* [ ] CI is passing.

### CODE REVIEW

**Comment by rafaelfranca** [comment]
Thank you for the pull request but the code is correct. The bracket is open to list the options and closed after the link. I think this is mostly a bug in the hyperlink parser in your terminal that is recoginizing the bracket as part of the link. On VSCode terminal, iterm and Terminal.app this link can be open without error.

File: railties/lib/rails/generators/rails/app/app_generator.rb [hunk 1]
```diff
@@ -264,7 +264,7 @@ class AppGenerator < AppBase
       class_option :api, type: :boolean, desc: "Preconfigure smaller stack for API only apps"
       class_option :minimal, type: :boolean, desc: "Preconfigure a minimal rails app"
       class_option :javascript, type: :string, aliases: ["-j", "--js"], default: "importmap", desc: "Choose JavaScript approach [options: importmap (default), webpack, esbuild, rollup]"
-      class_option :css, type: :string, aliases: "-c", desc: "Choose CSS processor [options: tailwind, bootstrap, bulma, postcss, sass... check https://github.com/rails/cssbundling-rails]"
+      class_option :css, type: :string, aliases: "-c", desc: "Choose CSS processor [options: tailwind, bootstrap, bulma, postcss, sass... check https://github.com/rails/cssbundling-rails"
```
Comment (rafaelfranca) [inline railties/lib/rails/generators/rails/app/app_generator.rb:267]:
```suggestion
      class_option :css, type: :string, aliases: "-c", desc: "Choose CSS processor [options: tailwind, bootstrap, bulma, postcss, sass] check https://github.com/rails/cssbundling-rails for more options"
```

**Comment by rafaelfranca** [comment]
Let's fix this in another way.

**Comment by creaturenex** [comment]
Awesome, I just tested in VSCode and it works like you said. Thanks for the suggestion!

File: .gitignore [hunk 2]
```diff
@@ -17,3 +17,6 @@ pkg/
 /tmp/
 /yarn-error.log
 /test-reports/
+
+# local plugin
+.history/
```
Comment (rafaelfranca) [inline .gitignore:22]: This should not be here. Maybe in your global gitignore.

**Comment by rafaelfranca** [comment]
Thank you

**Comment by creaturenex** [comment]
Thank you so much @rafaelfranca!

**Pull request merged**